bookkeeper-server/src/main/java/org/apache/bookkeeper/client/LedgerCreateOp.java

```diff
@@ -222,7 +222,7 @@ private void createComplete(int rc, LedgerHandle lh) {
         private int builderEnsembleSize = 3;
         private int builderAckQuorumSize = 2;
         private int builderWriteQuorumSize = 2;
-        private byte[] builderPassword;
+        private byte[] builderPassword = new byte[0];
```

Comment:
Better to have a constant, so we save an allocation

Reply:
Sure, will update tomorrow, and check the tests too.